tests/views/test_blacklist.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from rest_framework import status
+from rest_framework.reverse import reverse
+
+from rest_framework_jwt.blacklist.models import BlacklistedToken
+from rest_framework_jwt.serializers import JSONWebTokenAuthentication
+
+
+def test_only_user_can_blacklist_own_token(
+    user, staff_user, create_authenticated_client
+):
+    assert not BlacklistedToken.objects.exists()
```

Comment:
Each test has a clean slate (or should have, but in this library it _has_), which means that this assertion is redundant.